src/main/java/model/FigureSupplier.java

```diff
@@ -3,12 +3,13 @@
 import java.util.Random;
 
 public class FigureSupplier {
+    static final int randomLimit = 101;
+    static final int parametrsCount = 3;
+    static final int[] finalInt = new int[parametrsCount];
```

Comment:
```suggestion
    static final int RANDOM_LIMIT= 101;
    static final int PARAMETERS_COUNT = 3;
    static final int[] finalInt = new int[parametrsCount]; // this can be non constant
```